lib/config.js

```diff
@@ -16,35 +16,49 @@ const Config = {
   schema: {
     autocomplete: {
       title: 'Enable Autocomplete',
+      includeTitle: false,
```

Comment:
I'll try to get a PR merged in Atom after I'm done publishing an update for IJavascript.